project/RELEASE-PROCESS.md

```diff
@@ -0,0 +1,72 @@
+# Docker Release Process
+
+This document describes how regularly the Docker project is released. The Docker
+project release process targets the Engine, Compose, Kitematic, Machine, Swarm,
+Distribution, Notary and their underlying dependencies (libnetwork, libkv,
+etc...).
+
+Details of the process are described in 
+[RELEASE-CHECKLIST.md](https://github.com/docker/docker/blob/master/project/RELEASE-CHECKLIST.md).
+
+## Release cycle
+
+The Docker project follows a **time-based release cycle** and ships every nine
+weeks. A release cycle starts the same day the previous release cycle ends.
+
+The first six weeks of the cycle are dedicated to development and review. During
+this phase, new features and bugfixes submitted to any of the projects are
+**eligible** to be shipped as part of the next release. No changeset submitted
+during this period is however guaranteed to be merged for the current release
+cycle.
+
+## The freeze period
+
+Six weeks after the beginning of the cycle, the codebase is officially frozen
+and the codebase reaches a state close to the final release. A Release Candidate
+(RC) gets shipped at the same time. The freeze period is used to find bugs and
+get feedback on the state of the RC before the release. No features are merged
+during this period. As bugs are fixed, new RCs are shipped. This period lasts
```

Comment:
Instead of `No features....`, how about:
`During this freeze period, while the 'master' branch will continue its normal development cycle, no new features are merged into the RC. As bugs are fixed in 'master' the release owner will selectively 'cherry-pick' critical ones to be included into the RC.  As the RC changes, new ones are made available for the community to test and review.`